team/tech/coordination.md

```diff
@@ -2,6 +2,22 @@
 
 As a totally distributed team, it is important to have team practices that give one another insight into what we're working on, identify opportunities to collaborate, and un-block one another. This page describes some of the practices that we adopt towards these goals.
 
+
+(coordination:team-syncs)=
+## Weekly team syncs
+
+Every Monday, the 2i2c Hub Operations team conducts a team sync to get on the same page and coordinate their work for the week.
+
+1. **Create an issue using [the team sync issue template](https://github.com/2i2c-org/team-compass/issues/new?assignees=&labels=team-sync&template=team-sync.md)**. This issue is our space to discuss, update, and sync.
+2. **Team members give their responses**. You can copy/paste the template, and then give your responses in comments to the issue. You shouldn't feel forced to add content if you can't think of anything, use it as much as is useful.
+3. **Discuss and agree on next steps for 48hrs**. The goal is to ensure that important issues have somebody paying attention to them, and that team members are supported in the goals they work towards. After 48 hours, we'll close the issue.
```

Comment:
If you are talking here about the issue you opened for the sync, accordingly to the cron jobs listed above, I think you are actually closing it after 24+8 hs (I could be wrong, of course, since we are dealing with time shifts :wink:).
Since you automated the closing of the issue, I guess that block of time is only to agree on the management of the urgent matter, right? The alternative is to keep it open until the urgent matter is managed AND resolved...

Reply:
Good point - fixed with slightly more vague and thus more correct language :-)